security/advancedtls/examples/credential_reloading_from_files/client/main.go

```diff
@@ -89,22 +89,22 @@ func main() {
 		log.Fatalf("advancedtls.NewClientCreds(%v) failed: %v", options, err)
 	}
 
+	// Make a connection using the credentials.
+	ctx, _ := context.WithTimeout(context.Background(), defaultConnTimeout)
+	conn, err := grpc.DialContext(ctx, address, grpc.WithTransportCredentials(clientTLSCreds))
```

Comment:
So, related to the earlier conversation.  If you don't use `grpc.WithBlock`, then a context is not needed, as connection management happens behind the scene.  `WithBlock` makes `Dial[Context]` block until there is a connection.  I would delete `ctx` (also I'm surprised not capturing the `cancel` didn't give a lint error*), and use `grpc.Dial` instead.

\* - maybe `go lint` is ignoring anything not in the main module.  We might need to update `vet.sh` to `go lint` submodules if that is the case.  Would you mind looking into this in a follow-up PR?

Reply:
Sounds good. I've updated the code.

> * - maybe go lint is ignoring anything not in the main module. We might need to update vet.sh to go lint submodules if that is the case. Would you mind looking into this in a follow-up PR?
Sure, I can look into that. I ran `golint` several times locally, but it didn't produce anything even if I blatantly made some style errors in my code. Probably my lint version is different than the one used in presubmit tests. 
I think I saw `golint` or `govet` reporting errors in `advancedtls` before. But to make sure, let me create an issue to keep track of this effort:   https://github.com/grpc/grpc-go/issues/4075